src/EventListener/MapperListener.php

```diff
@@ -19,57 +20,22 @@
 use Symfony\Component\HttpKernel\Event\ViewEvent;
 
 /**
- * Class MapperListener.
- *
  * Applies the Dto\View dtoMapper to the by the controller action returned data, so a DTO (or array of DTOs) goes to
  * the response.
- *
- * @package Shopping\ApiTKDtoMapperBundle\EventListener
  */
 class MapperListener
 {
-    /**
-     * @var bool
-     */
-    private $masterRequest = true;
-
-    /**
-     * @var ControllerListener
-     */
-    private $controllerListener;
-
-    /**
-     * @var Reader
-     */
-    private $reader;
-
-    /**
-     * @var ArrayHelper
-     */
-    private $arrayHelper;
-
-    /**
-     * @var ContainerInterface
-     */
-    private $container;
-
-    /**
-     * @param ControllerListener $controllerListener
-     * @param Reader             $reader
-     * @param ArrayHelper        $arrayHelper
-     * @param ContainerInterface $container
-     */
-    public function __construct(ControllerListener $controllerListener, Reader $reader, ArrayHelper $arrayHelper, ContainerInterface $container)
-    {
-        $this->controllerListener = $controllerListener;
-        $this->reader = $reader;
-        $this->arrayHelper = $arrayHelper;
-        $this->container = $container;
+    private bool $masterRequest = true;
```

Comment:
Official naming according to symfony/http-foundation would be `$mainRequest`. I think we should adapt this for new code if this does not make problems

Reply:
Changed in https://github.com/byWulf/apitk-dtomapper-bundle/pull/15/commits/fc7beeab27802ceefe2b647da7eba92de8e8ebbb